src/transactions/ExtendFootprintTTLOpFrame.cpp

```diff
@@ -168,6 +174,9 @@ ExtendFootprintTTLOpFrame::doCheckValid(SorobanNetworkConfig const& config,
         if (!isSorobanEntry(lk))
         {
             innerResult().code(EXTEND_FOOTPRINT_TTL_MALFORMED);
+            mParentTx.pushSimpleDiagnosticError(
+                appConfig, SCE_STORAGE, SCEC_INVALID_INPUT,
+                "only entries with TTL can have it extended", {});
```

Comment:
I don't think this is a clear error the the user, if I saw this I'd google how to create a TTL entry. What about "only soroban entries can extend TTL" or "only contract data and code entries can extend TTL".